src/app_service/service/stickers/service.nim

```diff
@@ -28,6 +26,7 @@ import ../eth/dto/edn_dto as edn_helper
 
 # TODO Remove those imports once chat is refactored
 import status/statusgo_backend/chat as status_chat
+import status/types/[sticker]
```

Comment:
@jrainville what about this line, it still point to `staus-lib` type. You most likely forgot to remove it since we already have that as `StickerPackDto` on the `status-desktop` side.

Reply:
It's still needed for now, since we need the `Sticker` type. That type is used when we `sendStickerMessage`. This will all get refactored when that part of chat is refactored.